.travis.yml

```diff
@@ -3,18 +3,11 @@ env:
   - VERIFY_USE_PUBLIC_BINARIES=true
 jdk:
   - oraclejdk8
-  - oraclejdk9
-  - oraclejdk10
   - openjdk8
-  - openjdk9
   - openjdk10
```

Comment:
Can we drop 10 now?

Reply:
I'd imagine so if the plan is to go to 11 from 8. Not sure what benefit it brings still running against 10. I'll get that confirmed and then look to remove them.

Reply:
It's worth keeping 10 until 11 is working...